app-kotlin/src/main/kotlin/au/com/gridstone/trainingkotlin/screens/home/HomeViewModel.kt

```diff
@@ -16,16 +17,21 @@ import retrofit2.Response
 
 sealed class HomeViewState {
   object Loading : HomeViewState()
+  object Refreshing : HomeViewState()
```

Comment:
whats the difference between these two states?

Reply:
Loading check for loading view the progress bar , and  refresh link to swipe to refresh for testing purpose we need this

Reply:
do we need to make that distinction? Loading is _just_ loading right? I think we should keep them the same state. For tests, we can assert the same way

Reply:
Yes in the start I did that way but for testing, I was confused to test both separately but now it is clear now